CONTRIBUTING.md

```diff
@@ -14,6 +14,16 @@ flutter doctor -v
 This will list out all components used for Flutter development, and if they are installed. This should include either Android or iOS development (depending on your platform), as well as any desktop platform you want to target.  
 You can ignore the Chrome/Web component, since Finamp is not a web app.
 
+### NixOS and Flakes
+
+The project includes a `flake.nix` file that can help you install flutter and rust dependencies. The recommended way of using it is to
+do `nix develop .#fenix` (or you can use `dotenv` with `.envrc` file). 
```

Comment:
I created it mainly for myself and as such it is not entirely maintained, so I won't suggest it for new contributors from the start. Also there's no .envrc file in the repository (except in gitignore) so I didn't understood what you mean.

```suggestion
The project includes a `flake.nix` file that can help you install flutter and rust dependencies. The recommended way of using it is to
use `nix develop`, which allows rustup for Discord RPC dependency. Also there's `nix develop .#fenix` which stubs rustup with fenix, but it is a hack.
```

Reply:
Makes sense, should I move it somewhere in the bottom of the file?